images/pegasus/Dockerfile

```diff
@@ -0,0 +1,142 @@
+ARG BASE_APP_IMAGE
+
+# hadolint ignore=DL3006
+FROM ${BASE_APP_IMAGE}
+
+ARG DEBIAN_FRONTEND=noninteractive
+# see: https://github.com/AppImage/AppImageKit/wiki/FUSE#docker
+ENV APPIMAGE_EXTRACT_AND_RUN=1
+
+# Install prereqs
+RUN \
+    echo "**** Install Prereqs (Mesa/Vulkan/Fuse/QT/Misc) ****" && \
+		apt-get update && \
+		apt-get install -y software-properties-common gpg-agent wget p7zip-full libvulkan1 mesa-vulkan-drivers libglu1-mesa-dev qtbase5-dev qt5-qmake ffmpeg x11-xserver-utils libdbus-1-3 libgtk-3-0 libegl1 libsdl2-2.0-0 && \
```

Comment:
`mesa-vulkan-drivers` and `libvulkan1` should already be in `base-app`. It's probably best to remove those two here.

Reply:
I can remove and test.

Reply:
I just looked through the code...`mesa-vulkan-drivers` is available, but `libvulkan1` is not. I'm fixing this in `es-de` as well.

Reply:
I won't mess with the base app.